hsrc_lib/Database/LMDB/Raw.hsc

```diff
@@ -1298,13 +1298,19 @@ r_cursor_put rc =
     _throwLMDBErrNum "mdb_cursor_put" rc
 {-# INLINE r_cursor_put #-}
 
-mdb_cursor_put' :: MDB_WriteFlags -> MDB_cursor' -> MDB_val -> MDB_val -> IO Bool
-mdb_cursor_put' wf crs key val =
-    withKVPtrs key val $ \ pKey pVal ->
-    _mdb_cursor_put' (_crs_ptr' crs) pKey pVal wf >>= \ rc ->
-    r_cursor_put rc
+mdb_cursor_put' :: MDB_WriteFlags -> MDB_cursor' -> Ptr MDB_val -> Ptr MDB_val -> IO Bool
```

Comment:
Aren't other packages parts of the code or downstream clients that depend on `mdb_cursor_put` and `mdb_cursor_put'`. We're changing their types and this might break compatibility. What about leaving these as they were, and instead use a name like `mdb_cursor_put_val` instead?

Reply:
You are right, my change is too much churn: I've changed the types back, and the pointer-version is now called `mdb_cursor_put_ptr`

Reply:
Perfect!